docs/DEPLOYMENT.md

```diff
@@ -0,0 +1,206 @@
+node itself. This is called an *installer* image of EVE-OS. It is a separate
+binary that you can get from the EVE-OS distribution.
+
+There's nothing special about the *installer* image of EVE-OS. In fact, under the
+hood, it is simply a live image of EVE-OS that runs a single application: a tiny
+script that writes a live image (and since installer image is a live image it
+effectively writes itself) onto the storage device that is inaccessible from
+the outside of the edge-node. The right way to think about installer image is
+that it is a live image that is only ever booted once on a given edge-node. Once
+installer image is done and shuts the edge node down the live image is available
+on the storage device and there's no need to run installer image anymore.
+
+In general, EVE-OS assumes a hands-free environment and can't rely on a human
+operator to configure initial settings "at the console". Given that the installer
+image is nothing but EVE-OS, it too, needs to be told ahead of time about any
+configuration necessary. To find out more how EVE-OS gets configured follow
+[configuration](CONFIG.md) documentation. For the rest of this document we will
+assume either config partition having required configuration or dynamic configuration
+done by supplying overrides to the boot process of the installer image. 
+
+## Serial vs. soft serial numbers
+
+EVE controller has to be able to recognize EVE for the first time. This is done
+by relying on a piece of semi-unique information called a ```serial number```.
+The problem with serial numbers is that they come in many flavors and in cases
+where you don't have physical access to the edge node you may not even know
+the serial number ahead of time. This is why EVE-OS has a secondary serial
+number called ```soft serial```. Whenever you run EVE-OS installer image you
+can either tell it directly what do you want your soft serial number to be by
+passing an alpha-numeric sring in the ```eve_soft_serial``` variable OR you
+can rely on the installer image to generate a unique one for you. In the later
+case the only problem is getting that number back. EVE-OS installer image
+always prints a soft serial number at the end of the run and if the image
+is running from a writable media (like a USB stick) it will also deposit
+it in the INVENTORY partition as a newly created folder (the folder name is
+the soft serial number). 
+
+## Deploying EVE-OS in physical environments
+
+Physical environments assume an actual, physical edge node, but do not necessarily
+assume physical access to the edge node. In cases where physical access is not available
+it is common to rely on either iPXE booting or remote management solutions such
+as [HPE iLO](https://en.wikipedia.org/wiki/HP_Integrated_Lights-Out) or [Dell DRAC](https://en.wikipedia.org/wiki/Dell_DRAC). 
+
+With a few exceptions (like a Raspberry Pi or SBCs in general) a physical edge node
```

Comment:
I think it makes sense to exand the SBC acronym here to not loose the reader.

Reply:
fixed